src/backend/test.rs

```diff
@@ -916,4 +986,786 @@ mod tests {
         let mut backend = TestBackend::new(10, 2);
         backend.flush().unwrap();
     }
+
+    #[test]
+    #[cfg(feature = "scrolling-regions")]
+    fn scroll_region_up_whole_screen_by_0() {
+        let mut backend = TestBackend::new(10, 5);
```

Comment:
Consider putting this in a rstest fixture to remove one line from each of the tests.